supplements/curse-of-strahd/items.xml

```diff
@@ -161,4 +161,28 @@
 			<set name="attunement">true</set>
 		</setters>
 	</element>
+	<element name="Lost Sword" type="Magic Item" source="Curse of Strahd" id="ID_WOTC_COS_MAGIC_ITEM_LOST_SWORD">		
+		<description>
+			<p>An unnamed sentient shortsword.</p>
+			<p class="indent">You gain a +1 bonus to attack and damage rolls made with this weapon.</p>
+			<p class="indent">The sword continually sheds bright light in a 15-foot radius and dim light for an additional 15 feet. Only by destroying the sword can this light be extinguished.</p>
+			<p class="indent">A lawful good creature can attune itself to the sword in 1 minute.</p>
+			<p class="indent">While attuned to the weapon, the sword’s wielder can use the sword to cast the <em>crusader’s mantle</em> spell. Once used, this property of the sword can’t be used again until the next dawn.</p>
+			<p class="indent"><strong><em>Sentience.</em></strong> The sword is a lawful good weapon with an Intelligence of 11, a Wisdom of 13, and a Charisma of 13. It has hearing and normal vision out to a range of 120 feet. The weapon communicates by transmitting emotion to the creature carrying or wielding it.</p>
+			<p class="indent"><strong><em>Personality.</em></strong> The sword’s purpose is to fight evil.</p>
+		</description>
+		<setters>
+			<set name="category">Magic Weapons</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="type" addition="shortsword">Weapon</set>
+			<set name="rarity">Rare</set>
+			<set name="enhancement">1</set>
+			<set name="weapon">Shortsword</set>
+			<set name="attunement">true</set>
+		</setters>
+		<rules>
+			<grant type="Spell" id="ID_PHB_SPELL_CRUSADERS_MANTLE" level="3" spellcasting="Paladin" prepared="true" />
```

Comment:
You probably should remove `spellcasting="Paladin"` from the spell

Reply:
Done. Also fixed a type in the Description for the Sun Sword.